packages/battery/battery_linux/example/lib/main.dart

```diff
@@ -0,0 +1,78 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'dart:async';
+
+import 'package:battery/battery.dart';
```

Comment:
The battery_linux plugin should really use/test battery_linux directly; this is something I'm trying to move toward for all federated plugins (at least on desktop) going forward; duplicating main-plugin tests doesn't help since those will be run anyway, and there's no need to create circular dependencies. The platform interface implementation should provide a clear API surface to write examples and tests against.

Reply:
Done.